### DISCUSSION TITLE
Monthly rotation.

### DISCUSSION THREAD

**Discussion by jeffpanc** [edtechre/pybroker#128] [Q&A] [answered]
Hi, I am trying (wanting) to do a monthly rotation strategy.  Is there a way to rank and rotate at month's end? Thanks.

**Comment by edtechre** [comment, marked as answer]
Hi @jeffpanc,

Have you seen this notebook on rotational trading?
https://www.pybroker.com/en/latest/notebooks/10.%20Rotational%20Trading.html

You can check [ctx.dt](https://www.pybroker.com/en/latest/reference/pybroker.context.html#pybroker.context.ExecContext.dt) in ``rotate`` to only run your trading rules at the end of the month.

**Reply by jeffpanc** [reply]
Thanks I have been playing with the rotational code but wasn’t aware of .dt of the ctx.

**Reply by edtechre** [reply]
See https://www.pybroker.com/en/latest/reference/pybroker.context.html#pybroker.context.ExecContext.dt
